### PR TITLE
Remove engine.Table from more daemon side stuff

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis dug@us.ibm.com

### CODE REVIEW

**Comment by duglin** [comment]
ping @tiborvass @LK4D4

File: integration/api_test.go [hunk 1]
```diff
@@ -793,12 +794,14 @@ func TestDeleteImages(t *testing.T) {
 		t.Fatalf("%d OK expected, received %d\n", http.StatusOK, r.Code)
 	}
 
-	outs := engine.NewTable("Created", 0)
-	if _, err := outs.ReadListFrom(r2.Body.Bytes()); err != nil {
+	delImages := []types.ImageDelete{}
+	err = json.Unmarshal(r2.Body.Bytes(), &delImages)
```
Comment (LK4D4) [inline integration/api_test.go]: How about `json.NewDecoder(r2.Body)` instead?

Comment (duglin) [inline integration/api_test.go]: @LK4D4 when I do that I get:
./api_test.go:798: cannot use r2.Body (type *bytes.Buffer) as type []byte in argument to json.Unmarshal
so I think I need to use Bytes() to get to the actual bytes.  Unless I'm missing something.

Comment (LK4D4) [inline integration/api_test.go]: Ah, actually this is in tests, then forget it.

**Comment by LK4D4** [comment]
LGTM

**Comment by estesp** [comment]
LGTM

**Pull request merged**